readme.md

```diff
@@ -23,6 +23,8 @@ Usage:
 Options:
     -a                    Track current tmux session
     -A                    Track pane within current tmux session
+    -o [index]            Overwrite tracked index with current session in tmux session
```

Comment:
The documentation here should read 
`Replace bookmark at index with current session`